config/v1/types_operatorhub.go

```diff
@@ -6,10 +6,18 @@ import (
 
 // OperatorHubSpec defines the desired state of OperatorHub
 type OperatorHubSpec struct {
+	// disableAllDefaultSources allows you to disable all the default hub
+	// sources. If sources is specified, the configuration there will take
+	// precedence.
```

Comment:
Add an example: `if this is true, a specific sources entry could turn one back on.  If this is false, a specific sources entry could turn on off` or some such.